librz/core/linux_heap_glibc.c

```diff
@@ -1132,6 +1135,8 @@ RZ_API RzList *GH(rz_heap_tcache_list)(RzCore *core, GHT m_arena, MallocState *m
 		return tcache_list;
 	}
 	if (!GH(tcache_read)(core, tcache_start, rz_tcache)) {
+		GH(tcache_free)
+		(rz_tcache);
```

Comment:
Single line?

Reply:
`clang-format` automatically makes it multi line